connector/src/test/scala/com/basho/riak/spark/rdd/TimeSeriesTest.scala

```diff
@@ -160,4 +168,55 @@ class TimeSeriesTest extends AbstractRDDTest {
       "{time:111555, user_id:'ratman',temperature_k:3502.212}" +
       "]", stringify(data))
   }
+
+  @Ignore("Ignored because of lack getting schema support in Riak TS")
+  @Test
+  def sqlRangeQuery(): Unit = {
+    val from = tsRangeStart.getTimeInMillis - 5
+    val to = tsRangeEnd.getTimeInMillis + 10
+    val bucketName =  DEFAULT_TS_NAMESPACE.getBucketTypeAsString
+
+    val sqlContext = new RiakSQLContext(sc, DEFAULT_TS_NAMESPACE.getBucketTypeAsString)
+    val dfTemplate: DataFrame = sqlContext.sql(
+      s"SELECT time, user_id, temperature_k " +
+        s" FROM $bucketName " +
+        s" WHERE time > $from AND time < $to AND surrogate_key = 1 AND family = 'f'")
+
+    dfTemplate.registerTempTable("ts_test")
+
+    val df = sqlContext.sql("SELECT time, user_id, temperature_k FROM ts_test")
+    val data = df.collect()
+
+    assert(data != null)
+  }
+
+  @Test
+  def dataFrameGenericLoad(): Unit = {
+    val from = tsRangeStart.getTimeInMillis - 5
+    val to = tsRangeEnd.getTimeInMillis + 10
+    val sqlContext = new SQLContext(sc)
+    val df = sqlContext.read
+      .format("org.apache.spark.sql.riak")
+      .schema(
+        StructType(
+          List(StructField("surrogate_key", LongType, nullable=false),
+            StructField("family", StringType, nullable=false),
+            StructField("time", TimestampType, nullable=false),
+            StructField("user_id", StringType, nullable=false),
+            StructField("temperature_k", DoubleType, nullable=true)))
+      )
+      .load(DEFAULT_TS_NAMESPACE.getBucketTypeAsString)
+      .filter(s"time > CAST($from AS TIMESTAMP) AND time < CAST($to AS TIMESTAMP) AND surrogate_key = 1 AND family = 'f'")
+
+    df.printSchema()
+    val data = df.toJSON.collect()
+
+    assertEqualsUsingJSONIgnoreOrder("[" +
```

Comment:
Timestamps when converted to strings may have different values depending on timezone, which makes test fail.